CMakeLists_files.cmake

```diff
@@ -451,6 +453,7 @@ list( APPEND PUBLIC_HEADER_FILES
       opm/common/OpmLog/OpmLog.hpp
       opm/common/OpmLog/StreamLog.hpp
       opm/common/OpmLog/TimerLog.hpp
+      opm/common/utility/ActiveGridCells.cpp
```

Comment:
Should be the header

Reply:
Nice catch thanks. did cause jenkins to fail. Fixed.